client/src/components/Login/loginForm/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import image from './user.png';
+import { Icon } from 'antd';
+import './style.css'
+
+export default class Login extends Component {
+  state = {
+    username: '',
+    password: '',
+    loginError: false,
+  }
+
+  handleChange = ({ target: { name, value } }) => {
+    this.setState({ [name]: value });
+  }
+
+  login = (event) => {
+    event.preventDefault();
+    const { password, username } = this.state;
+    if (!password.trim() || !username.trim()) {
+      this.setState({ loginError: true });
+    }
+    //fetch//
+  }
+
+  render() {
+    return (
+      <div className='login__form__container'>
```

Comment:
the proper tag would be `section`
also, use BEM methodology correctly `login__container`